### PR TITLE
feat(file-utils): add a utility package for handling file uploads

### PR DESCRIPTION
**Pull request by akshatdubeysf**
## Description

- added a package for convenient uploading of files and multipart requests in general
- added relevant tests
- update documentation

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## Checklist:

- [X] Performed a self-review of my own code
- [X] npm test passes on your machine
- [X] New tests added or existing tests modified to cover all changes
- [X] Code conforms with the style guide
- [X] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/file-utils/package.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
+{
+  "name": "@sourceloop/file-utils",
+  "version": "0.0.0",
+  "description": "A Tenant guard extension to prevent cross tenant access and operations",
```
Comment (samarpanB) [inline packages/file-utils/package.json:4]: change this

File: packages/file-utils/README.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
+# File Utils
+
+This package provides utilities for adding file upload and validations in LoopBack 4 microservices.
+
+## Installation
+
+To install the package, run the following command:
+
+```shell
+npm install @loopback/file-utils
```
Comment (samarpanB) [inline packages/file-utils/README.md:10]: fix this

Comment (akshatdubeysf) [reply packages/file-utils/README.md]: fixed

File: packages/file-utils/package.json (hunk 1 above)
Comment (akshatdubeysf) [reply packages/file-utils/package.json]: fixed

**Pull request merged**